modules/benchmark/R/load.data.R

```diff
@@ -16,6 +16,7 @@ load.data <- function(data.path, format, start_year = NA, end_year=NA, site=NA,
   require(PEcAn.benchmark)
   require(lubridate)
   require(udunits2)
+  require(dplyr)
```

Comment:
@bpbond, any comments on `::` vs. `NAMESPACE` import? Prefer the former to keep it explicit? I don't have enough experience developing packages to have formed any opinion one way or the other, but that's my guess.